### PR TITLE
Add CIBIR Support to SecNetPerf

### PR DESCRIPTION
**Pull request by nibanks**
## Description

Adds a new option `-cibir:<bytes>` to secnetperf for specifying the CIBIR extension's well-known identifier.

## Testing

Tested by manually running over localhost.

### CODE REVIEW

File: src/perf/lib/HpsClient.cpp [hunk 1]
```diff
@@ -72,7 +72,8 @@ HpsClient::Init(
     }
 
     const char* target;
-    if (!TryGetValue(argc, argv, "target", &target)) {
+    if (!TryGetValue(argc, argv, "target", &target) &&
+        !TryGetValue(argc, argv, "server", &target)) {
```
Comment (thhous-msft) [inline src/perf/lib/HpsClient.cpp:76]: Why this change? We don't specify "server" is allowed anywhere in the help, and this seems completely unrelated.

Comment (nibanks) [reply src/perf/lib/HpsClient.cpp:76]: This is a improvement for my personal well-being. 😄 I can never remember the arg name, and I always end up using `-server`. I don't feel the need to clutter the help, but I'd like to support it.

**Review by thhous-msft — APPROVED** [review]

File: src/perf/lib/ThroughputClient.cpp [hunk 2]
```diff
@@ -288,6 +299,31 @@ ThroughputClient::StartQuic()
             &LocalIpAddr);
     }
 
+    if (CibirIdLength) {
+        BOOLEAN Opt = TRUE;
+        Status =
+            MsQuic->SetParam(
```
Comment (thhous-msft) [inline src/perf/lib/ThroughputClient.cpp:305]: Isn't the whole point of this change to not require sharing the UDP binding? I thought that with the parameter set, all short headers would get a cid, which we're trying to avoid iirc.

**Review by thhous-msft — APPROVED** [review]
